Jenkinsfile_k8s

```diff
@@ -185,6 +186,7 @@ pipeline {
           PKR_VAR_agent_os_version      = "${env.agent_type.split('-')[1]}"
           PKR_VAR_architecture          = "${env.cpu_architecture}"
           PKR_VAR_image_type            = "${env.compute_type}"
+          PKR_VAR_azure_gallery_subscription_id = "${env.AZURE_GALLERY_SUBSCRIPTION_ID}"
```

Comment:
```suggestion
```

Removing since we deployed https://github.com/jenkins-infra/azure/pull/560 and https://github.com/jenkins-infra/azure/pull/561 (new galleries in new subscription in us east 2)